src/java.base/share/classes/java/lang/reflect/AccessFlag.java

```diff
@@ -403,7 +425,7 @@ private static class LocationToFlags {
                                        BRIDGE, VARARGS, NATIVE,
                                        ABSTRACT, STRICT, SYNTHETIC)),
                           entry(Location.INNER_CLASS,
-                                Set.of(PUBLIC, PRIVATE, PROTECTED,
+                                Set.of(PUBLIC, PRIVATE, PROTECTED, IDENTITY, VALUE, PRIMITIVE,
```

Comment:
I also caught this issue in the previous version.  Glad that it's fixed in this version but the locations of these new constant should include INNER_CLASS as well.